### PR TITLE
Added the bootstrap css framework and refactored to remove the show page.

### PR DESCRIPTION
**Pull request by trippykyle**
Some useful CSS resources are at [W3C](https://www.w3.org/Style/CSS/learning)
Trying to style all pages using the latest [bootstrap](https://getbootstrap.com/docs/5.3/getting-started/introduction/) release.
Also, remove the [jbuilder](https://github.com/rails/jbuilder) logic from the app and [refactor](https://en.wikipedia.org/wiki/Code_refactoring) to remove show view.

### CODE REVIEW

File: Gemfile
Comment (francoisa) [inline Gemfile:1]: Remove jbuilder from the file with 'bundle remove'

File: Gemfile [hunk 1]
```diff
@@ -59,3 +59,7 @@ group :test do
   gem "capybara"
   gem "selenium-webdriver"
 end
+
+gem "bootstrap", "~> 5.3"
```
Comment (francoisa) [inline Gemfile]: Move these above the `group` blocks.

File: app/views/tasks/edit.html.erb [hunk 2]
```diff
@@ -1,10 +1,10 @@
-<h1>Editing task</h1>
-
+<h3>
+Editing
```
Comment (francoisa) [inline app/views/tasks/edit.html.erb]: indentation is off or put on one line

File: app/views/tasks/edit.html.erb [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+</h3>
 <%= render "form", task: @task %>
 
 <br>
 
 <div>
-  <%= link_to "Show this task", @task %> |
-  <%= link_to "Back to tasks", tasks_path %>
+<%= link_to "Tasks", tasks_path, {class: 'btn btn-success btn-sm'} %>
```
Comment (francoisa) [inline app/views/tasks/edit.html.erb]: indentation

File: app/views/tasks/index.html.erb [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
-    <p>
-      <%= link_to "Show this task", task %>
-    </p>
-  <% end %>
-</div>
+    <div id="tasks">
+      <% @tasks.each do |task| %>
+        <div id="<%= dom_id task %>">
+  
+          <strong>Content:</strong>
```
Comment (francoisa) [inline app/views/tasks/index.html.erb]: not needed.

**Comment by francoisa** [comment]
the form partial is not styled using bootstrap.

**Review by francoisa — CHANGES_REQUESTED** [review]
See my comments.

**Comment by trippykyle** [comment]
> the form partial is not styled using bootstrap.

not sure where to find the partial form

File: app/views/tasks/edit.html.erb [hunk 5]
```diff
@@ -1,10 +1,9 @@
-<h1>Editing task</h1>
+<h3>Editing</h3>
 
-<%= render "form", task: @task %>
+    <%= render "form", task: @task %>
```
Comment (francoisa) [inline app/views/tasks/edit.html.erb:3]:
```suggestion
<%= render "form", task: @task %>
```

File: app/views/tasks/edit.html.erb [hunk 6]
```diff
@@ -1,10 +1,9 @@
...
 
-<br>
+    <br>
```
Comment (francoisa) [inline app/views/tasks/edit.html.erb:5]:
```suggestion
<br>
```

**Comment by francoisa** [comment]
> > the form partial is not styled using bootstrap.
> 
> not sure where to find the partial form
the [partial](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) I was referring to is:  **app/views/tasks/_form.html.erb**

**Review by francoisa — CHANGES_REQUESTED** [review]

**Review by francoisa — APPROVED** [review]
Looks good to me

**Pull request merged**